index.js

```diff
@@ -0,0 +1,13 @@
+const express = require("express");
+const app = express();
+const port = 5000;
+const doetenv = require("dotenv");
+doetenv.config();
+const getDailyWeather = require("./APIs/getDailyWeather");
```

Comment:
There should be a seperate index.js file inside APIs folder this should be referenced from there